monkey/tests/unit_tests/monkey_island/cc/services/zero_trust/scoutsuite/test_scoutsuite_auth_service.py

```diff
@@ -5,7 +5,10 @@
 
 from common.config_value_paths import AWS_KEYS_PATH
 from monkey_island.cc.database import mongo
-from monkey_island.cc.server_utils.encryptor import get_encryptor, initialize_encryptor
+from monkey_island.cc.server_utils.encryption.data_store_encryptor import (
+    get_encryptor,
+    initialize_encryptor,
```

Comment:
Maybe these should be renamed to `get_datastore_encryptor()` and `initialize_datastore_encryptor()`